main.sh

```diff
@@ -190,9 +193,9 @@ main() {
   if [[ "${deploymentMode}" == "What-if" ]]; then
     info "Previewing deployment with what-if"
     if [[ "${deploymentScope}" == "sub" ]]; then
-      whatIfCmdBase="az deployment sub what-if --location ${location} --template-file ${workingDir}/${templateFile}"
+      whatIfCmdBase="az deployment sub what-if --location ${location} --template-file ${workingDir}"
```

Comment:
Add a comment that workingDir is pointing to the templatefile

Reply:
change `workingDir="${LOCAL_IAC_SOURCE_CODE_DIR}"` to `templatefile ="${LOCAL_IAC_SOURCE_CODE_DIR}"`

<img width="596" alt="Screenshot 2025-03-21 at 11 38 57" src="https://github.com/user-attachments/assets/5bd01239-317b-46de-8112-39db3dd32d9e" />